src/Mock/Data.js

```diff
@@ -0,0 +1,29 @@
+import curry from '../components/images/curry.jpg';
```

Comment:
Images are consider to be an asset for the app so please make a directory called assets and keep images there ..